ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -131,6 +131,13 @@ private class SqlLedger(
       SourceQueueWithComplete[Long => PersistenceEntry],
       SourceQueueWithComplete[Long => PersistenceEntry]) = createQueues()
 
+  checkpointQueue
+    .watchCompletion()
+    .foreach(_ => logger.warn("checkpoint queue has been closed!"))(DEC)
```

Comment:
Is it really a warning though? For example, when shutting down the sandbox, this is just a normal event that happens.

Reply:
good point! The thing is that it's only valuable if it fails with an error. I'll look if there is a callback for that.

Reply:
I addressed this.